SKETCH.md

```diff
@@ -632,6 +646,7 @@ Type: select all that apply (optional)
 - I was using Elm and wanted to go further down the rabbit hole.
 - I was using Idris and wanted to go further up the rabbit hole.
 - I am a Haskeller and am comfortably situated in the rabbit hole but was looking to make a shift to webdev.
+- I am a JS/TS developer and I'm tired of stuff breaking all the time.
```

Comment:
I feel like this "I'm tired of stuff breaking all the time," is somewhat disrespectful towards such developers.